src/cache_padded.rs

```diff
@@ -164,6 +191,8 @@ mod test {
         assert_eq!(x.1, 37);
     }
 
+    // An alignment of 64 is only guaranteed on nightly.
+    #[cfg(feature = "nightly")]
```

Comment:
We don't need this attribute. The test doesn't verify alignment - it only verifies the distance between adjacent `CachePadded`s.

Reply:
Right, but that distance depends on the alignment of CachePadded - if it's a wierd alignment it might be greater than 64 (happened when I was running tests). I could modify the test to check `dist >= 64`.

Reply:
Well, dinstance depends on alignment *and* size of `CachePadded`.
Yes, checking for `dist >= 64` is fine as well.